internal/fields/model.go

```diff
@@ -44,82 +43,6 @@ type ReusableConfig struct {
 	TopLevel bool `yaml:"top_level"`
 }
 
-func (orig *FieldDefinition) Update(fd FieldDefinition) {
```

Comment:
I didn't find any usages of this `Update` function around the code.

It looks like that it was removed in this PR: https://github.com/elastic/elastic-package/pull/1335/files#diff-77cc155ee037ac620b1e188302f60fdf736637a25e5021e76dbad83594edd09cL133

Reply:
Interesting, this used to be neccesary :thinking: But good if we don't need it anymore.

Reply:
At least, that function is not called anymore...